src/server/http/xsrf.js

```diff
@@ -10,6 +11,10 @@ export default function (kbnServer, server, config) {
       return reply.continue();
     }
 
+    if (whitelist.includes(req.path)) {
```

Comment:
It seems to be the most restrictive way, like people should be very specific about particular URL that is supposed to be white-listed.

Reply:
++ agreed, I like this approach — I think it needs to be this specific. It's definitely better to have people list multiple urls in the config than to try to be smart here, if possible.